cli/tests/coverage/complex.ts

```diff
@@ -10,7 +10,8 @@ export interface Complex {
   baz: string;
 }
 
-export function complex(
+// 패딩에 대한 더 많은 문자.
```

Comment:
I guess these have to do something with figuring out ranges, care to add a comment explaining that?

Reply:
Went overly commenty on it, which is fine since the comments are for line padding, ensuring comments don't show up as uncovered and they add some wide characters.